classification/models/condition_text_matching.py

```diff
@@ -1063,6 +1063,28 @@ def condition_matching_suggestions(ct: ConditionText, ignore_existing=False) ->
     return suggestions
 
 
+def raise_condition_resolution_flag(classification: Classification, condition_resolution):
```

Comment:
I think we can tidy things up by changing this to
```
def apply_condition_resolution(classification: Classification, new_condition_resolution: ConditionResolved):
```
and as well as raising the change flag, the method can also call
```
classification.condition_resolution = new_condition_resolution
classification.save(update_fields=["condition_resolution"])
```
and then removing the calls to classification.save from outside

and finally the method could be marked with an atomic transaction - to ensure we get both the flag and the updated classification or neither.